src/ninja.cc

```diff
@@ -932,7 +953,7 @@ std::string EvaluateCommandWithRspfile(const Edge* edge,
   return command;
 }
 
-void printCompdb(const char* const directory, const Edge* const edge,
+void printOneCompdbObject(std::string const &directory, const Edge* const edge,
```

Comment:
nit: `PrintOneCompdbObject`.

Use `git clang-format` to reformat your commits, the `&` goes on the side of the type. See `CONTRIBUTING.md` for the link to the style guide.